app/helpers/host_aspects_helper.rb

```diff
@@ -0,0 +1,29 @@
+module HostAspectsHelper
+  # override host's tabs to add all aspects as tabs
+  def host_additional_tabs(host)
```

Comment:
this is used not just in UI views, but also to render responses to API calls, where 'tabs' sounds confusing.